src/utils/schema/updateWidgetProperties.ts

```diff
@@ -37,12 +41,14 @@ type PropertyUpdaters = {
 };
 
 const propertyUpdaters: PropertyUpdaters = {
+  description: updateDescription,
```

Comment:
According to my comment in widgetPropertyUpdaters :
```suggestion
  description: updateProperty('description'),
```

Reply:
I did some fix to avoid it, let me know if you had another idea